frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/EditTaskRecommendedItems/NewNameRecommendation.tsx

```diff
@@ -0,0 +1,70 @@
+import { useValidateBpmnTaskId } from '../../../../hooks/useValidateBpmnId';
+import { useBpmnContext } from '../../../../contexts/BpmnContext';
+import { StudioIconTextfield, StudioRecommendedNextAction } from '@studio/components';
+import { KeyVerticalIcon } from '@studio/icons';
+import type Modeling from 'bpmn-js/lib/features/modeling/Modeling';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface NewNameRecommendationProps {}
+
+export const NewNameRecommendation = ({}: NewNameRecommendationProps): React.ReactElement => {
```

Comment:
No need to specify props type for components without props. 😊 
```suggestion
export const NewNameRecommendation = (): React.ReactElement => {
```